src/components/user/PersonalProfileEditForm.js

```diff
@@ -44,6 +55,17 @@ export default function PersonalProfileEditForm() {
       });
     } else {
       try {
+        // If there's a profile picture, upload it first
+        if (formData.profilePicture) {
+          const formDataImage = new FormData();
+          formDataImage.append('profile', formData.profilePicture);
+
+          // Make the API call to upload the image
+          const imageResponse = await axiosInstance.post(
```

Comment:
The linter warns us that there is no need to assign the result to a variable if we're not going to use the variable. We can make the linter happy by just adding an underscore at the front of the variable:

```suggestion
          const _imageResponse = await axiosInstance.post(
```

or we can remove the variable altogether (the axios POST function would still be called, we're just not storing the value anywhere):

```suggestion
          await axiosInstance.post(
```

Reply:
Thank you so much❤️, I didn't know that by adding underscore at the front of the variable, we can resolve this issue